### PR TITLE
Single article view

### PR DESCRIPTION
**Pull request by lukeperrin10**
pt: https://www.pivotaltracker.com/story/show/178911977

## User Story 
```
As a User
To read piece of news that interests me
I want to be able to to do it when clicking on News Item
```
## Suggested Changes 

- Adds single article view
- adds route for single article through: info/news/articles/:id
- adds api call for single article 

## Video 

https://user-images.githubusercontent.com/79849155/127498158-5a806681-8ba3-4321-a886-051073175d29.mp4

https://user-images.githubusercontent.com/79849155/127498167-662df73b-b4c4-47e0-8f29-07717e9ae243.mp4

https://user-images.githubusercontent.com/79849155/127498174-a113812d-6913-4257-84d0-d91069e3654e.mp4

### CODE REVIEW

File: cypress/integration/mobile/visitorCanNavigateUsingHeader.feature.js [hunk 1]
```diff
@@ -1,59 +0,0 @@
...
-      "Mui-selected"
-    );
-  });
-
-  it("is expected to navigate to about home page", () => {
-    cy.get("[data-cy=home-tab]").click();
-    cy.url().should("include", "/home");
-    cy.get("[data-cy=home-tab]").should("have.class", "Mui-selected");
-  });
-});
```
Comment (1CIM) [inline cypress/integration/mobile/visitorCanNavigateUsingHeader.feature.js:59]: Why is this removed?

**Review by 1CIM — COMMENTED** [review]
Nice job guys! just one comment.

File: cypress/integration/mobile/visitorCanNavigateUsingHeader.feature.js (hunk 1 above)
Comment (lukeperrin10) [reply cypress/integration/mobile/visitorCanNavigateUsingHeader.feature.js:59]: Creates a failing test that would be out of scope for us to fix. All tests then go green without that. Though it best that that test be fixed in correct corresponding chore/feature

Comment (SevaDer14) [reply cypress/integration/mobile/visitorCanNavigateUsingHeader.feature.js:59]: Yes this test checks ability to navigate desktop header. So it really not made for mobile. When hamburger or whatever responsive stuff added it should be replaced with corresponding test but mobile specific.

Comment (1CIM) [reply cypress/integration/mobile/visitorCanNavigateUsingHeader.feature.js:59]: okay just thought the name of the test would be the same just changing the content of the test

**Review by 1CIM — APPROVED** [review]
